.gitignore

```diff
@@ -1 +1 @@
-.ds.store
+.ds_store
```

Comment:
Note sure if this has to be case sensitive: .DS_Store

Reply:
it works and has removed the file from git so looks like it is not.